examples/sigstore-go-verification/README.md

```diff
@@ -1,9 +1,51 @@
+# Building examples
+
+To build example programs, run `make build-examples` in the repo root.
+The built executables will be in the `examples/` subdirectory:
+```shell
+$ make build-examples && ls -tr examples | tail -3
+go build -C ./examples/oci-image-verification -o oci-image-verification .
+go build -C ./examples/sigstore-go-signing -o sigstore-go-signing .
+go build -C ./examples/sigstore-go-verification -o sigstore-go-verification .
+
+$ find examples -type f -perm -u+x | sort
+examples/oci-image-verification/oci-image-verification
+examples/sigstore-go-signing/sigstore-go-signing
+examples/sigstore-go-verification/sigstore-go-verification
+```
+
+# oci-image-verification
+
+This is a CLI fo verifying signatures on the OCI images. View the help text with `-h` or `--help` for all the options.
```

Comment:
```suggestion
This is a CLI for verifying signatures on the OCI images. View the help text with `-h` or `--help` for all the options.
```

Reply:
fixed in [e1cf27b](https://github.com/sigstore/sigstore-go/pull/412/commits/e1cf27b285a88801e5e506cd1365e9a0e3a264bd)